src/test/controllers/users.controller.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+import sampleUser from '../fixtures/loggedInUserData';
+import '../../server/models/users.model';
+
+/*
+ * Test for users.controller.
+ */
+
+let assert = require('assert');
+let should = require('should');
+let user = require('mongoose').model('user');
+
+describe('Test for users.controller', function() {
+  describe('#getMentorList()', function() {
+    it('should create a new User with fake session', function (done) {
+      var u = sampleUser.loggedInUserData;
```

Comment:
Do not use var: http://programmers.stackexchange.com/questions/274342/is-there-any-reason-to-use-the-var-keyword-in-es6